Facehugger inhands, xeno tweaks, and monkey bugfixes

phil235 opened the pull request:
Some of these changes are salvaged from #8629
- Aliens now use the same examine as monkeys. Monkey and aliens now all use carbon/examine(). (relevant to #3574)
- Fixes monkeys being able to strip and uncuff people while handcuffed themselves. Fixes #4735
- Add an inhand sprite for facehuggers (taken from Xhuis) (but they don't appear for large queens, it'd look ugly).
- Xenos now regenerate plasma on alien weed even when injured but at half the normal rate. They also heal their clone damage now.
- Moved queen/large/update_icons() to humanoid/update_icons.dm
- Alien humanoids and monkeys can now use "me".
- Alien hivemind chat now uses <span class='alien'> (dark purple). Also the alien queen name now appears bigger in the hivemind chat.

phil235 committed:
Monkey and aliens now all use carbon/examine().
Fixes monkeys being able to strip and uncuff people while handcuffed themselves.
Add an inhand sprite for facehuggers (taken from Xhuis) (but they don't appear for large queens, it'd look ugly).
Xenos now regenerate plasma on alien weed even when injured but at half the normal rate.
Moved queen/large/update_icons() to humanoid/update_icons.dm

RemieRichards commented on code/modules/mob/living/carbon/alien/organs.dm:
```diff
@@ -79,6 +79,7 @@
 			var/mod = 1
 			if(!isalien(owner))
 				mod = 0.2
+			owner.adjustPlasma(plasma_rate/2)
```
```*0.5``` instead please, same effect, slightly cheaper.

WJohn commented:
:+1: x :100:  Thank you thank you thank you.

phil235 committed:
Simplifies plasmavessel/on_life() a bit, and make xeno on weeds also heal clone damage.

phil235 committed:
Alien humanoids and monkeys can now use "me".
Alien hivemind chat now uses <span class='alien'> (dark purple). Also the alien queen name now appears bigger in the hivemind chat.

daman997 commented:
/me puts on his suit and xeno mask

bgobandit commented:
"Alien humanoids and monkeys can now use "me"."

are you sure about this

do you know what you are unleashing

(aliens getting around the whisper power, that's what. I bet you were thinking about xeno ERP you pervert)

WJohn commented:
Shhh ignore those two nothing could POSSIBLY go wrong.

Merge this please.

melandor0 commented:
Aliens used to have /me

Amy Lessen.

cyfause commented:
amy time
